### PR TITLE
[Kotlin Migration] DrawableContainer 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] DrawableContainer

### CODE REVIEW

File: AnkiDroid/src/main/java/com/wildplot/android/rendering/DrawableContainer.kt [hunk 1]
```diff
@@ -13,63 +13,39 @@
...
-
-import java.util.Vector;
+package com.wildplot.android.rendering
 
+import android.annotation.SuppressLint
+import com.wildplot.android.rendering.graphics.wrapper.GraphicsWrap
+import com.wildplot.android.rendering.interfaces.Drawable
+import java.util.Vector
 
 @SuppressLint("NonPublicNonStaticFieldName")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/wildplot/android/rendering/DrawableContainer.kt:24]: Again: I believe this can be removed

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Less sure on this one

File: AnkiDroid/src/main/java/com/wildplot/android/rendering/DrawableContainer.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/wildplot/android/rendering/DrawableContainer.kt]: Done, PTAL.

**Review by david-allison-1 — APPROVED** [review]
LGMT

**Review by Arthur-Milchior — APPROVED** [review]

**Pull request merged**